src/web/src/app/space/model.ts

```diff
@@ -7,3 +7,29 @@ export class Space {
 export interface SpaceListResponse {
 	items: Space[];
 }
+
+export interface SpaceInterface {
+	id: string;
+	name: string;
+	key: string;
+	requestsAllowed: boolean;
+	createdAt: Date;
+}
```

Comment:
Это та же модель что и Space. Нужно старую удали, а эту переименовать в Space.
```suggestion
export interface Space {
	id: string;
	name: string;
	key: string;
	requestsAllowed: boolean;
	createdAt: Date;
}
```

Reply:
Готово.